medusa/server/web/home/handler.py

```diff
@@ -1793,6 +1793,7 @@ def updateShow(self, show=None):
 
         # force the update
         try:
+            show_obj.remove_images()
```

Comment:
See one line change.

Reply:
Point is, that this is an object oriented approach. As the logic to delete the files in now in the ImageClass, where it should be. To make it more clean, we could even replace some of logic for cleaning up the images after a show delete, using this method.